registry/macos_ui.test

```diff
@@ -1,7 +1,7 @@
 contact=groovinchip@gmail.com
 
 fetch=git clone https://github.com/GroovinChip/macos_ui.git tests
-fetch=git -C tests checkout f1053fc31958dbaa362a0a4c8288e8cf0919ab8c
+fetch=git -C tests checkout 8c250edb0b1015a9db9ef09033f0d5ab15a144ce
```

Comment:
@GroovinChip am I holding this right? It looks like the docs for macos_ui say there is a special branch for flutter/tests.

Reply:
Yes, I haven't had a chance to update that branch yet. It's called `customer_testing`. I'm AFK and it doesn't look like GH mobile will let me open a PR; I'll try and get to it later this evening, or tomorrow morning at the latest.

Reply:
No worries at all, just clarifying. So this commit should be the commit on the customer_testing branch, not dev?

Reply:
Yep. Basically for every stable macos_ui release, I [am supposed to] checkout the customer_testing branch, switch to flutter master, merge in the latest commits from the `dev` branch, make changes as needed, and push. 

So actually I'm wrong about a PR lol, that wouldn't be enough. Although as I write this it strikes me that I think the CI on that branch runs on master, so perhaps a PR would do, in that the CI would fail if there's anything that needs to be updated.

Reply:
Thank you!